denonavr/ir_code.py

```diff
@@ -0,0 +1,507 @@
+MODE_AUTO_MANUAL = 'RCKSK0410156'
+MEMORY = 'RCKSK0410160'
+RDS_EU_only_SEARCH = 'RCKSK0410163'
+RT_EU_only = 'RCKSK0410166'
+PTY_EU_only = 'RCKSK0410169'
+CURSOR_UP_TUNER = 'RCKSK0410170'
+CURSOR_DOWN_TUNER = 'RCKSK0410171'
+CURSOR_LEFT_TUNER = 'RCKSK0410173'
+CURSOR_RIGHT_TUNER = 'RCKSK0410174'
+CURSOR_ENTER_TUNER = 'RCKSK0410175'
+PRESET_1__Numeric_1 = 'RCKSK0410176'
+PRESET_2__Numeric_2 = 'RCKSK0410177'
+PRESET_3__Numeric_3 = 'RCKSK0410178'
+PRESET_4__Numeric_4 = 'RCKSK0410179'
+PRESET_5__Numeric_5 = 'RCKSK0410180'
+PRESET_6__Numeric_6 = 'RCKSK0410181'
+PRESET_7__Numeric_7 = 'RCKSK0410182'
+PRESET_8__Numeric_8 = 'RCKSK0410183'
+Numeric_9 = 'RCKSK0410184'
+Numeric_0 = 'RCKSK0410185'
+RETURN_TUNER = 'RCKSK0410241'
+SHIFT_A = 'RCKSK0410248'
+SHIFT_B = 'RCKSK0410249'
+SHIFT_C = 'RCKSK0410250'
+SHIFT_D = 'RCKSK0410251'
+SHIFT_E = 'RCKSK0410252'
+SHIFT_F = 'RCKSK0410253'
+SHIFT_G = 'RCKSK0410254'
+CURSOR_UP = 'RCKSK0470027'
+CURSOR_DOWN = 'RCKSK0470028'
+CURSOR_LEFT = 'RCKSK0470029'
+CURSOR_RIGHT = 'RCKSK0470030'
+ENTER = 'RCKSK0470031'
+RETURN = 'RCKSK0470034'
+JPEG_Skip = 'RCKSK0470144'
+JPEG_Skip = 'RCKSK0470145'
+PRESET_CH_UP = 'RCKSK0470146'
+PRESET_CH_DOWN = 'RCKSK0470147'
+PRESET_CH_SHIFT = 'RCKSK0470148'
+MEMORY = 'RCKSK0470160'
+PAGE_SEARCH_CHARA_SEARCH = 'RCKSK0470163'
+PRESET_1 = 'RCKSK0470176'
+PRESET_2 = 'RCKSK0470177'
+PRESET_3 = 'RCKSK0470178'
+PRESET_4 = 'RCKSK0470179'
+PRESET_5 = 'RCKSK0470180'
+PRESET_6 = 'RCKSK0470181'
+PRESET_7 = 'RCKSK0470182'
+PRESET_8 = 'RCKSK0470183'
+SHIFT_A = 'RCKSK0470248'
+SHIFT_B = 'RCKSK0470249'
+SHIFT_C = 'RCKSK0470250'
+SHIFT_D = 'RCKSK0470251'
+SHIFT_E = 'RCKSK0470252'
+SHIFT_F = 'RCKSK0470253'
+SHIFT_G = 'RCKSK0470254'
+PLAY = 'RCKSK0470773'
+PAUSE = 'RCKSK0470774'
+STOP = 'RCKSK0470775'
+SKIP_PLUS = 'RCKSK0470776'
+SKIP_MINUS = 'RCKSK0470777'
+REPEAT_ONE = 'RCKSK0470778'
+REPEAT_OFF = 'RCKSK0470779'
+RANDOM_ON = 'RCKSK0470780'
+RANDOM_OFF = 'RCKSK0470781'
+REPEAT_ALL = 'RCKSK0470785'
+PRESET_1_Numeric_1 = 'RCKSK0740001'
+PRESET_2_Numeric_2 = 'RCKSK0740002'
+PRESET_3_Numeric_3 = 'RCKSK0740003'
+PRESET_4_Numeric_4 = 'RCKSK0740004'
+PRESET_5_Numeric_5 = 'RCKSK0740005'
+PRESET_6_Numeric_6 = 'RCKSK0740006'
+PRESET_7_Numeric_7 = 'RCKSK0740007'
+PRESET_8_Numeric_8 = 'RCKSK0740008'
+Numeric_9 = 'RCKSK0740009'
+Numeric_0 = 'RCKSK0740010'
+MEMORY = 'RCKSK0740012'
+PRESET_CH_SHIFT = 'RCKSK0740013'
+DIRECT_SEARCH = 'RCKSK0740018'
+PRESET_CH_DOWN = 'RCKSK0740021'
+PRESET_CH_UP = 'RCKSK0740022'
+BAND_AM_FM = 'RCKSK0740023'
+MODE = 'RCKSK0740024'
+TUNING_UP = 'RCKSK0740025'
+TUNING_DOWN = 'RCKSK0740026'
+ENTER = 'RCKSK0740066'
+CURSOR_UP = 'RCKSK0740069'
+CURSOR_DOWN = 'RCKSK0740070'
+CURSOR_LEFT = 'RCKSK0740071'
+CURSOR_RIGHT = 'RCKSK0740072'
+RETURN = 'RCKSK0740073'
+SHIFT_A = 'RCKSK0740096'
+SHIFT_B = 'RCKSK0740097'
+SHIFT_C = 'RCKSK0740098'
+SHIFT_D = 'RCKSK0740099'
+SHIFT_E = 'RCKSK0740100'
+SHIFT_F = 'RCKSK0740101'
+SHIFT_G = 'RCKSK0740102'
+BAND_FM = 'RCKSK0740153'
+BAND_AM = 'RCKSK0740154'
```

Comment:
Please provide a valid docstring for this .py file and convert the constants to upper case only, that they fit the python naming conventions.